clang/test/CodeGen/aarch64-sve2p1-intrinsics/acle_sve2p1_qcvtn.c

```diff
@@ -0,0 +1,78 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
```

Comment:
Hmm, I could only see aarch64_sve_sqcvtn_x2 tests here, can you add aarch64_sve_sqcvtn_x4 as well?

Reply:
oh, sorry, I see now.